ftw/redirector/browser/excel_views.py

```diff
@@ -0,0 +1,108 @@
+from ftw.redirector import _
+from ftw.redirector.config import RulesValidator
+from ftw.redirector.excel import create_rules_excel
+from ftw.redirector.excel import load_rules_from_excel
+from ftw.redirector.excel import RULES_START_ROW
+from ftw.redirector.interfaces import IRedirectConfig
+from ftw.redirector.interfaces import IRedirectorLayer
+from plone.directives import form
+from plone.formwidget.namedfile.widget import NamedFileWidget
+from plone.namedfile.field import NamedBlobFile
+from plone.namedfile.interfaces import INamedFileField
+from Products.Archetypes.utils import contentDispositionHeader
+from Products.Five.browser import BrowserView
+from Products.statusmessages.interfaces import IStatusMessage
+from StringIO import StringIO
+from z3c.form import button
+from z3c.form.interfaces import IFieldWidget
+from z3c.form.validator import WidgetValidatorDiscriminators
+from z3c.form.widget import FieldWidget
+from zope.component import adapter
+from zope.component import provideAdapter
+from zope.interface import implementer
+from zope.interface import Invalid
+
+
+class ExportRedirectConfigView(BrowserView):
+
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
+
+    def __call__(self):
+        response = self.request.RESPONSE
+        filename = "redirector_config.xlsx"
+
+        disp = contentDispositionHeader('attachment', 'l1', filename=filename)
```

Comment:
Why `l1`?

I'd prefer to not use an archetypes function here, since the package is dexterity and archetypes this is the only dependency to archetypes.
It also very easy to that explicitly:
```python
response.setHeader('Content-Disposition', 'attachment; filename={}'.format(filename))
```